manifests/repos.pp

```diff
@@ -28,7 +28,7 @@
         apt::source { 'kubernetes':
           location => pick($kubernetes_apt_location,'http://apt.kubernetes.io'),
           repos    => pick($kubernetes_apt_repos,'main'),
-          release  => pick($kubernetes_apt_release,"kubernetes-${codename}"),
+          release  => pick($kubernetes_apt_release,'kubernetes-xenial'),
```

Comment:
Instead of hardcoding the value can we just assign the value to a variable and use the variable.

Reply:
What's the benefit of assigning it to a variable if its not used in any other place?